### DISCUSSION TITLE
what does Quick Save do?

### DISCUSSION THREAD

**Discussion by diekhans** [KarchinLab/open-cravat#244] [Q&A]
It is not obvious what Quick Save does or how to load it.  I can't find anything it the documentation.

Also, if Firefox/Mac the text that is display has no padding around it, making it hard to read.

**Comment by kmoad** [comment]
Quicksave saves the current filter and widget layout as the default, so that the jobs is shown in that state the next time it's loaded. I've updated the quicksave mouseover text to better describe it.

https://github.com/KarchinLab/open-cravat/commit/87cd2ac4a9bd1195cc97d89fba0c590c28f4514e
